### PR TITLE
Update layout to cope with bitwarden #2615

### LINKED ISSUE #2615: Update layout to cope with bitwarden
## What went wrong? 😲
 
Coming from [this PR](https://github.com/msupply-foundation/health-supply-hub/pull/808)

If BW injects an area at the top of the page, then the logout button is shunted off the bottom of the screen

## Expected behaviour 🤔
Page should adjust and not lose the bottom of the LH menu

To fix, in index.html change
```
<body>
    <div id="root"></div>
  </body>
```
to 

```
<body>
    <div id="content-row">
      <div id="outer-wrapper">
        <div id="inner-wrapper">
          <div id="root"></div>
        </div>
      </div>
    </div>
  </body>
```

and change the global styles in `Viewport.tsx` to

```
  body: {
    // displaying as table (and introducing the below wrappers around #root)
    // makes page responsive to injected popups like the Bitwarden one
    display: 'table',
    height: '100vh',
    width: '100%',
  },
  '#content-row': {
    display: 'table-row',
    height: '100%',
  },
  '#outer-wrapper': {
    display: 'table-cell',
    height: '100%',
  },
  '#inner-wrapper': {
    height: '100%',
    width: '100vw',
    position: 'relative',
    overflow: 'auto',
  },
  '#root': {
    width: '100vw',
    display: 'flex',
    flexDirection: 'column',
    position: 'absolute',
    top: '0',
    bottom: '0',
    left: '0',
    right: '0',
  },
```

## How to Reproduce 🔨

Steps to reproduce the behaviour:

## Your environment 🌱

 - Version:

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #2615

# 👩🏻‍💻 What does this PR do? 
Updates layout to cope with Bitwarden

# 🧪 How has/should this change been tested? 
- [ ] Have Bitwarden plugin installed for Firefox or Chrome
- [ ] Login and keep the Bitwarden popup at the top
- [ ] See that logout button is still there
- [ ] Can try in different views with explorer's developer tool

### CODE REVIEW

File: client/packages/common/src/hooks/useDeleteConfirmation/useDeleteConfirmation.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const useDeleteConfirmation = <T>({
     cantDelete,
     selectRows,
   } = messages;
-  const t = useTranslation('common');
+  const t = useTranslation();
```
Comment (roxy-dao) [inline client/packages/common/src/hooks/useDeleteConfirmation/useDeleteConfirmation.ts:33]: Sidetrack: Sorry, I've been fixing these as I come across them but thought I'll just go a Find + Replace 👀

File: client/packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/TabTables.tsx [hunk 2]
```diff
@@ -157,6 +157,7 @@ export const PricingTableComponent: FC<TableProps> = ({
       description: 'description.fc-sell-price',
       width: 100,
       align: ColumnAlign.Right,
+      // eslint-disable-next-line new-cap
```
Comment (roxy-dao) [inline client/packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/TabTables.tsx:160]: Sidetrack: Missed from currency PR

File: client/packages/host/public/index.html [hunk 3]
```diff
@@ -1,6 +1,9 @@
-<html>
-  <head> </head>
-  <body>
-    <div id="root"></div>
-  </body>
-</html>
+<body>
```
Comment (roxy-dao) [inline client/packages/host/public/index.html:1]: Fix here too..

**Review by lache-melvin — APPROVED** [review]
Looks great! Thanks for cleaning up the translation things too ~~~~~~~~~

**Pull request merged**